presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveFileIterator.java

```diff
@@ -130,8 +147,13 @@ private FileStatusIterator(Table table, Path path, FileSystem fileSystem, Direct
         {
             this.path = path;
             this.namenodeStats = namenodeStats;
+            this.fileStatusIterator = list(table, path, fileSystem, directoryLister);
+        }
+
+        private RemoteIterator<LocatedFileStatus> list(Table table, Path path, FileSystem fileSystem, DirectoryLister directoryLister)
```

Comment:
unnecessary change?

Reply:
Please elaborate. Method is used in the ctor of FileStatusIterator

Reply:
Yes, the commit extracts the method while it was "inline" in the ctor.
It seems like a redundant unrelated change, so please inline it back (unless i am missing something)

Reply:
From my perspective it improves readability - I can inline back (revert) this change if it's unnecessary

Reply:
OTOH it's a refactor, so would belong to a separate commit.

(I think it's redundant, so i'd prefer to inline, but no strong feeling about this.)